README.md

```diff
@@ -162,7 +194,35 @@ To cite Concrete in academic papers, please use the following entry:
   note={\url{https://github.com/zama-ai/concrete}},
 }
 ```
+### 🟨 Contributing 
+
+There are two ways to contribute to Concrete. You can:
+- [Open issues](https://github.com/zama-ai/concrete/issues/new/choose) to report bugs and typos, or to suggest new ideas
+- Request to become an official contributor by emailing hello@zama.ai.
+
+Becoming an approved contributor involves signing our Contributor License Agreement (CLA)). Only approved contributors can send pull requests (PRs), so get in touch before you do!
+<br></br>
+
+### 🟨 License
+This software is distributed under the **BSD-3-Clause-Clear** license. If you have any questions, please contact us at hello@zama.ai.
+<p align="right">
+  <a href="#table-of-contents" > ↑ Back to top </a> 
+</p>
 
-## License
+## Support
+
+<a target="_blank" href="https://community.zama.ai">
+  <img src="https://github.com/zama-ai/concrete-ml/assets/157474013/9d518f66-30da-4738-a154-a4d9fce93704">
```

Comment:
to me, this is too big image, as compared to the rest of the README

Reply:
<img width="1243" alt="Screenshot 2024-02-08 at 15 15 23" src="https://github.com/zama-ai/concrete/assets/157474013/db931405-183f-4333-b925-9730e65f1953">
<img width="1123" alt="Screenshot 2024-02-08 at 15 15 48" src="https://github.com/zama-ai/concrete/assets/157474013/0d778b75-653e-4690-b0ba-efd167aa2e69">

 
To compare